<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddToDatabaseForm</class>
 <widget class="QDialog" name="AddToDatabaseForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Speciality</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="specialityInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Group</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="groupInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surnameInput"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Book Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bookNumberInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
